hyperfunctions.cpp

```diff
@@ -207,13 +207,15 @@ void  HyperFunctions::DispClassifiedImage()
 void  HyperFunctions::DispFalseImage()
 {
    Mat temp_img;
+
    cv::resize(false_img,temp_img,Size(WINDOW_WIDTH, WINDOW_HEIGHT),INTER_LINEAR); 
    imshow("False Image", temp_img);
 }
 
 void  HyperFunctions::DispSpecSim()
 {
    Mat temp_img;
+   spec_simil_img.convertTo(spec_simil_img, CV_8UC1);
```

Comment:
conversion should be done in different location

Reply:
fixed